extensions/Persona/Config.pm

```diff
@@ -19,11 +19,6 @@ use constant REQUIRED_MODULES => [
         module  => 'JSON',
         version => 0,
     },
-    {
-        package => 'libwww-perl',
-        module  => 'LWP::UserAgent',
-        version => 0,
-    },
```

Comment:
https://bugzilla.mozilla.org/show_bug.cgi?id=1283138 is the bug where we talked about removing Persona. It was just turned off -- I'll have a bug filed to remove it (and fix tests) later but that's outside of the scope of this task.